src/main/java/propertyFiles/Bundle.properties

```diff
@@ -1219,7 +1219,18 @@ dataset.guestbookResponse.guestbook.responseTooLong=Please limit response to 255
 # dataset.xhtml
 dataset.configureBtn=Configure
 dataset.pageTitle=Add New Dataset
-dataset.editBtn=Edit
+
+dataset.accessBtn=Access Dataset
+dataset.accessBtn.header.download=Download Options
+dataset.accessBtn.header.explore=Explore Options
+dataset.accessBtn.header.compute=Compute Options
+dataset.linkBtn=Link Dataset
```

Comment:
There was already "dataset.link=Link Dataset" and "dataset.email.datasetContactBtn=Email Dataset Contact" as examples. Are these now gone / no longer used? Should we be cleaning these up?

Reply:
Good catch. Will make sure to review this as part of the final round of code clean up.

Reply:
Reviewed and resolved duplicates. Organized the bundle a bit to group msg's better. Will have an opportunity to revisit the link btn and msg text in issue #6772. There is ample opportunity to review and refactor the other 2,500+ lines of bundle text, as suggested in #4825.